sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_version.py

```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "4.4.1"
+VERSION = "4.5.0b1"
```

Comment:
Consider changing the development status classifier in setup.py while you're at it